include/wchar.h

```diff
@@ -57,6 +57,8 @@
  * Reference: Opengroup.org
  */
 
+#define _LIBCPP_WCHAR_H
```

Comment:
why add this

Reply:
let's change the search order in:
/home/xiaoxiang/vela/nuttx/tools/Config.mk

Reply:
I am not sure our seach order is necessarily wrong. For example:

```
arm-none-eabi-g++ -c -nostdinc++ -Wno-attributes -Wno-unknown-pragmas -Wno-psabi -std="gnu++20" -Os -fno-strict-aliasing -fomit-frame-pointer -fno-common -Wall -Wshadow -Wundef -ffunction-sections -fdata-sections -mlittle-endian -march=armv7e-m -mtune=cortex-m7 -mfpu=fpv5-sp-d16 -mfloat-abi=hard -mthumb -Wa,-mthumb -Wa,-mimplicit-it=always -isystem /home/trns1997/nuttxspace/nuttx/include/libcxx -isystem /home/trns1997/nuttxspace/nuttx/include -D__NuttX__ -DNDEBUG  -pipe -D_LIBCPP_BUILDING_LIBRARY -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I /home/trns1997/nuttxspace/nuttx/libs/libxx/libcxx/src -D__GLIBCXX__    libcxx/src/atomic.cpp -o  libcxx/src/atomic.o
```
We can see that here ` -isystem /home/trns1997/nuttxspace/nuttx/include/libcxx -isystem /home/trns1997/nuttxspace/nuttx/include` libcxx is included before our includes. So i am not sure what to change?

Reply:
if so, it's strange that libcxx can't find the self-header file first.